packages/credentials/src/status-list-credential.ts

```diff
@@ -1,47 +1,57 @@
 import pako from 'pako';
 import { getCurrentXmlSchema112Timestamp } from './utils.js';
 import { VerifiableCredential, DEFAULT_VC_CONTEXT, DEFAULT_VC_TYPE, VcDataModel } from './verifiable-credential.js';
-import type { ICredentialStatus} from '@sphereon/ssi-types';
 import { Convert } from '@web5/common';
 
 export const DEFAULT_STATUS_LIST_VC_CONTEXT = 'https://w3id.org/vc/status-list/2021/v1';
 export const DEFAULT_STATUS_LIST_VC_TYPE = 'StatusList2021Credential';
 
+/**
+ * The status purpose dictated by Status List 2021 spec.
+ * @see {@link https://www.w3.org/community/reports/credentials/CG-FINAL-vc-status-list-2021-20230102/#statuslist2021entry | Status List 2021 Entry}
+ */
 export enum StatusPurpose {
-  REVOCATION = 'revocation',
-  SUSPENSION = 'suspension',
+  revocation = 'revocation',
+  suspension = 'suspension',
 }
 
 /**
  * The size of the bitstring in bits.
  * The bitstring is 16KB in size.
  */
-const BITSTRING_SIZE = 16 * 1024 * 8; // 16KB in bits
+const BITSTRING_SIZE = 16 * 1024 * 8; // 16KiB in bits
 
 /**
  * StatusListCredentialCreateOptions for creating a status list credential.
  *
  * @param statusListCredentialId The id used for the resolvable path to the status list credential [String].
  * @param issuer The issuer URI of the credential, as a [String].
  * @param statusPurpose The status purpose of the status list cred, eg: revocation, as a [StatusPurpose].
- * @param issuedCredentials The credentials to be included in the status list credential, eg: revoked credentials, list of type [VerifiableCredential].
+ * @param revokedCredentials The credentials to be included in the status list credential, eg: revoked credentials, list of type [VerifiableCredential].
  */
 export type StatusListCredentialCreateOptions = {
   statusListCredentialId: string,
   issuer: string,
   statusPurpose: StatusPurpose,
-  issuedCredentials: VerifiableCredential[]
+  revokedCredentials: VerifiableCredential[]
 };
 
 /**
- * The StatusList2021Entry instance representing the core data model of a vc status list 2021.
+ * Credential status lookup information included in a Verifiable Credential that supports status lookup.
+ * Data model dictated by the Status List 2021 spec.
  *
- * @see {@link https://www.w3.org/community/reports/credentials/CG-FINAL-vc-status-list-2021-20230102/ | Status List 2021 Entry}
+ * @see {@link https://www.w3.org/community/reports/credentials/CG-FINAL-vc-status-list-2021-20230102/#example-example-statuslist2021credential | Status List 2021 Entry}
  */
-export interface StatusList2021Entry extends ICredentialStatus {
+export interface CredentialStatusModel {
```

Comment:
Following our spec,  and for clarity I think it should e called `StatusList2021Entry`

https://github.com/TBD54566975/web5-spec/blob/main/spec/vc.md

There is a different credStatusModel that will be for vc data model 2.0

Reply:
Reverted! I had strong opinion on this, but you win on this one! We can chat about it next week.